x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/SpatialPlugin.java

```diff
@@ -67,7 +69,14 @@
 import static java.util.Collections.singletonList;
 
 public class SpatialPlugin extends Plugin implements ActionPlugin, MapperPlugin, SearchPlugin, IngestPlugin, ExtensiblePlugin {
-   private final SpatialUsage usage = new SpatialUsage();
+    private final SpatialUsage usage = new SpatialUsage();
+
+    private final LicensedFeature.Momentary GEO_CENTROID_AGG_FEATURE =
+        LicensedFeature.momentary("spatial", "geo-centroid-agg", License.OperationMode.GOLD);
+    private final LicensedFeature.Momentary GEO_GRID_AGG_FEATURE =
+        LicensedFeature.momentary("spatial", "geo-grid-agg", License.OperationMode.GOLD);
+    private final LicensedFeature.Momentary GEO_LINE_AGG_FEATURE =
+        LicensedFeature.momentary("spatial", "geo-line-agg", License.OperationMode.GOLD);
```

Comment:
Might be a Nit: should the names use underscore instead of dash assuming we want them to match existing enums? That is, `geo_line_agg` instead of `geo-line-agg`?

Reply:
I've been using dashes for separation in the newer features, partly to distinguish with the old that were only using underscores because it was the literal name of the enum value.